src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
- [ ] This and 5 other tests are failing....Kindly make the tests run successfully to guarantee the considerable integrity or reliability of your code. :pray: 

![image](https://github.com/mohamedgamaleldin999999/metrics-app/assets/99715640/feca1d85-ded2-4af7-a1fd-e081ad13d2cb)